packages/esm-service-queues-app/src/queue-table/cells/queue-table-patient-age-cell.component.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import dayjs from 'dayjs';
+import { type QueueTableColumnFunction, type QueueTableCellComponentProps } from '../../types';
+
+export const QueueTableWaitTimeCell = ({ queueEntry }: QueueTableCellComponentProps) => {
+  const birthdate = dayjs(queueEntry.patient.person.birthdate);
+  const todaydate = dayjs();
+  const age = todaydate.diff(birthdate, 'years');
```

Comment:
I have seen logic around O3 that attempts to standardize some of this in places.  Not sure if we want to try to use that here, or if that should be a different column type.  The logic I've seen is something like "if the patient is < 1 month old, show age in days, if the patient is < 2 years old show age in months, else show age in years".  Don't rely on that exactly, but it's something like that.  @ibacher / @denniskigen do you remember?